src/bp-moderation/classes/suspend/class-bp-suspend-member.php

```diff
@@ -71,6 +71,9 @@ public function __construct() {
 		add_filter( 'get_avatar_url', array( $this, 'get_avatar_url' ), 9999, 3 );
 		add_filter( 'bp_core_fetch_avatar_url_check', array( $this, 'bp_fetch_avatar_url' ), 1005, 2 );
 		add_filter( 'bp_core_fetch_gravatar_url_check', array( $this, 'bp_fetch_avatar_url' ), 1005, 2 );
+		add_filter( 'bp_get_followers', array( $this, 'bb_exclude_moderated_user_ids' ), 9999, 1 );
```

Comment:
@bb-yudhisthir We should exclude users from queries. I think that is the optimized way. This way will create an issue if the hidden user list is very long.